src/engine/tools/engine-suite/src/engine_router/cmds/eps_update.py

```diff
@@ -0,0 +1,49 @@
+import os
+from google.protobuf.json_format import ParseDict
+
+from api_communication.client import APIClient
+import api_communication.proto.router_pb2 as erouter
+import api_communication.proto.engine_pb2 as engine
+
+
+def run(args):
+
+    # Get the params
+    if args['refresh-interval'] < 0:
```

Comment:
Please skip this, as only API calls should be made and not parameter values ​​should be checked. This is already done in the backend:
```
void checkSettings(uint eps, uint intervalSec)
{
if (eps < 1)
{
throw std::runtime_error("EPS Limit must be greater than 0");
}

if (intervalSec < 1)
{
throw std::runtime_error("EPS Interval must be greater than 0");
}
}
```

Reply:
No, I can't skip this, because we use google protobuf. the eps and the interval are defined as unsigned integers (A protobuf type and not native to python), so to build the request I can only use positive numbers. It is not possible to create a request with numbers less than zero (Exception out of range).